backend/tests/apps/slack/models/google_auth_test.py

```
@@ -0,0 +1,311 @@
+        self, mock_get_or_create, mock_save, mock_refresh
+    ):
+        """Test authenticate with existing expired token."""
+        # Create existing auth with expired token
+        existing_auth = GoogleAuth(
+            user=self.member,
+            access_token=self.valid_token,
+            refresh_token=self.valid_refresh_token,
+            expires_at=self.expired_time,
+        )
+
+        result = GoogleAuth.authenticate("http://auth.url", self.member)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)